docs/walkthrough/create-training-data.md

```diff
@@ -71,10 +87,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
 When reading in files constructed in this way, a unique label is generated for each pair. Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
 
 
+```
+
+```{warning}
+Please remove/replace comma in your data fields since csv use comma `,` as delimiter.
+
+
```

Comment:
```suggestion
```{warning}
Please remove/replace comma in your data fields if you are using a comma `,` as a delimiter.


```